Virtual Monopoly Deal/src/Main/Game.java

```diff
@@ -70,10 +75,43 @@ public void runGame(){
         
         System.out.println("Entering main game loop"); //for testing
         //main game loop.
+        int pcount = 0;
+        int bcount = 0;
+        boolean p_dealed = true;
+       
         while(true){
+           
             
-            //if turn = players:
-            playerTakeTurn();
+            if(playerTurn){
+                if(pcount == 0 && p_dealed == false && p_first_turn == false){
```

Comment:
should p_first_turn == true here in the if statement? Otherwise not sure if player will ever be dealt their 2 cards. Let me know if it worked on your end.

Reply:
no- once p_first_turn turns into false, it never turns true again, signifying its not the first turn & allowing the game to deal two cards at the beginning of the turns that aren't the first one. It works on my end :)